### PR TITLE
Merge ess-library into matrix-stack

### PR DESCRIPTION
**Pull request by gaelgatelement**
- Follow up of https://github.com/element-hq/ess-helm/pull/42 . Only matrix-stack exists now.

### CODE REVIEW

File: scripts/assemble_helm_charts_from_fragments.sh [hunk 1]
```diff
@@ -27,10 +27,7 @@ function assemble_helm_chart_from_fragments() {
 
 [ ! -d "$chart_root" ] && echo "$chart_root must be a directory that exists" && exit 1
 
-assemble_helm_chart_from_fragments "$chart_root"/ess-library
-
 for subchart in "$chart_root"/*/; do
-  [[ "$subchart" =~ /ess-library/?$ ]] && continue
   [[ "$subchart" =~ /matrix-stack/?$ ]] && continue
   assemble_helm_chart_from_fragments "$subchart"
 done
```
Comment (benbz) [inline scripts/assemble_helm_charts_from_fragments.sh]: This all can go

File: scripts/set_chart_version.sh [hunk 2]
```diff
@@ -25,10 +25,7 @@ function set_chart_version() {
 
 [ ! -d "$chart_root" ] && echo "$chart_root must be a directory that exists" && exit 1
 
-set_chart_version "$chart_root"/ess-library
-
 for subchart in "$chart_root"/*/; do
-  [[ "$subchart" =~ /ess-library/?$ ]] && continue
   [[ "$subchart" =~ /matrix-stack/?$ ]] && continue
   set_chart_version "$subchart"
 done
```
Comment (benbz) [inline scripts/set_chart_version.sh]: This can all go

File: scripts/helm_dependency_update_recursive.sh
Comment (benbz) [inline scripts/helm_dependency_update_recursive.sh:1]: This script can go

**Comment by gaelgatelement** [comment]
I'm forcing the merge, it's trying to run the CI from main branch.

**Pull request merged**